src/app/footer/footer.component.html

```diff
@@ -41,11 +41,11 @@ <h5>QUICK LINKS </h5>
         <div class="support-links">
           <h5>CUSTOMER SUPPORT</h5>
           <ul>
-            <li><a href="#">Morganti Luxury Boutique Displays</a></li>
+            <li><a routerLink="address">Morganti Luxury Boutique Displays</a></li>
             <li><a href="#">Sign in/My Account</a></li>
-            <li><a href="#">Privacy Policy</a> </li>
-            <li><a href="#">Terms&Conditions</a> </li>
-            <li><a href="#">Delivery&Returns</a> </li>
+            <li><a routerLink="privacy-policy">Privacy Policy</a> </li>
+            <li><a routerLink="term-condition">Terms&Conditions</a> </li>
+            <li><a routerLink="delivery-return" >Delivery&Returns</a> </li>
```

Comment:
will it work for every page?

For example, suppose we are on route http://localhost:8081/product, and clicking on any of the footer routes let say on the Privacy Policy link, In that case, it may redirect us to http://localhost:8081/product/privacy-policy
Please check and confirm this scenario